web/server/parser/packageParser.go

```diff
@@ -101,10 +101,12 @@ func (self *outputParser) registerTestFunction() {
 }
 func (self *outputParser) recordTestMetadata() {
 	testName := strings.Split(self.line, " ")[2]
-	self.test = self.testMap[testName]
-	self.test.Passed = !strings.HasPrefix(self.line, "--- FAIL: ")
-	self.test.Skipped = strings.HasPrefix(self.line, "--- SKIP: ")
-	self.test.Elapsed = parseTestFunctionDuration(self.line)
+	if test, ok := self.testMap[testName]; ok {
```

Comment:
:) Yes, you are correct, it was odd of me to submit an issue that I knew would break my own parser. I've known for some time that I would have to address it. Thanks for beating me to it!

https://trello.com/c/7a70yBOP/35-support-for-go-1-5